sdks/java/io/kafka/src/main/java/org/apache/beam/sdk/io/kafka/KafkaIO.java

```diff
@@ -847,6 +849,16 @@ public void validate() {
     }
   }
 
+  /**
+   * Limit on time it takes to initialize a partition in {@link UnboundedKafkaReader#start()},
+   * which involves fetching offset information the servers. Kafka consumer blocks forever
+   * in case of a network issue. We cancel the request after this timeout and throw an error.
+   */
+  private static final long PARTITION_INITIALIZATION_TIMEOUT_MS = 60 * 1000;
+  @VisibleForTesting
+  static final String PARTITION_INITIALIZATION_TIMEOUT_MS_CONFIG =
```

Comment:
I removed this config. Using 2 * request.timeout.ms config in Kafka if it is set by the user, other defaults to 1 minute.